recipes/arrow/all/conanfile.py

```diff
@@ -152,7 +152,7 @@ def requirements(self):
         if self.options.with_jemalloc:
             self.requires("jemalloc/5.3.0")
         if self.options.with_mimalloc:
-            self.requires("mimalloc/1.7.6")
+            self.requires("mimalloc/2.0.6")
```

Comment:
Do we really need to bump this version? Is it related to adding this new arrow version?

Reply:
I just bumped it to what's currently in use as it's a major version difference. But that version has been in use for years, so we should be save reverting this.